### DISCUSSION TITLE
Open Database with Windows Hello not only unlock

### DISCUSSION THREAD

**Discussion by dffvb** [keepassxreboot/keepassxc#11088] [Q&A] [answered]
Hi there, I see that I can unlock the database with Windows Hello but after every reboot I have to enter the passwort. Is there a way to open and unlock the database with Windows Hello?

**Comment by droidmonkey** [comment, marked as answer]
#7020

**Reply by dffvb** [reply]
Did not find this, but thank you for developping it, looking forward to 2.8.0
